src/main/drivers/pwm_output.c

```diff
@@ -126,7 +126,7 @@ static pwmOutputPort_t *pwmOutConfig(const timerHardware_t *timerHardware, uint8
 
 static void pwmWriteBrushed(uint8_t index, uint16_t value)
 {
-    *motors[index]->ccr = (value - 1000) * motors[index]->period / 1000;
```

Comment:
Should be fine as it is. Why we need this additional casting? Floating point math is faster on the F4?